### ISSUE TITLE
user profile generation question

### ISSUE THREAD

**Issue by HappyDay1998** [HKUDS/RLMRec#26]
Regarding to the user profiles generation part, how do you randmly choose 3 recipes for each user? In the whole recipe pool or just the recipes in the training set for the corresponding users? If I choose 3 recipes global randomly, will this have the risk of data leakage for the test set?  Thank you so much!

**Comment by Re-bin** [comment]
Hi 👋!

Thanks for your interest in RLMRec! Yes, for profile generation we sample only training-set interactions to avoid data leakage.

I hope the above answer is helpful to you :)

Best regards,
Xubin
